recipes/poliastro/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: poliastro
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/poliastro/poliastro.git
+  git_tag: v{{ version }}
+
+build:
+  number: 1
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - numpy >=1.9
+    - scipy
+    - numba >=0.23
+    - astropy >=1.0
+    - jplephem
```

Comment:
It can go in this PR or a new one. Both will become separate feedstocks either way. There might be build errors when they are released from the same PR. However, these can easily be addressed by restarting the builds.